### PR TITLE
Make show.fitCallbacks and show.history group certain metrics together

### PR DESCRIPTION
**Pull request by tafsiri**
This PR adds functionality to group accuracy metrics together and group loss metrics together. This closes https://github.com/tensorflow/tfjs/issues/812 but takes the approach of making the function to the appropriate thing rather than provide more specific configuration options.

the default output from show.fitCallbacks with the metrics `['loss', 'val_loss', 'acc', 'val_acc']` now shows the following.

screen shot 2018-11-02 at 2 06 50 pm

Calling show.history _at the end of training_ (i.e. looking at epochs) with the same metrics shows. 

screen shot 2018-11-02 at 2 06 56 pm

In general these methods will always try to put together `acc` metrics and `loss` metrics respectively. I think we have a small enough set of things that we can put the logic for what should be put together inside the function and just handle whatever metrics the user passes in.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-vis/33)

### CODE REVIEW

**Comment by tafsiri** [comment]
I will also likely follow up this PR with one to rename the x axis label to either batch or epoch and to allow the user to specify which callbacks they want metrics plotted for (currently hardcoded to onBatchEnd and onEpochEnd).

**Comment by tafsiri** [comment]
@caisq updated
